File: b1240di/app/src/main/java/com/example/b1240di/MainActivity.java
package com.example.b1240di;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.provider.Settings;
import android.widget.TextView;

public class MainActivity extends AppCompatActivity {

    private TextView textView;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        textView = (TextView)findViewById(R.id.textView);

        String id = Settings.Secure.getString(
                getContentResolver(),
                Settings.Secure.ANDROID_ID);

        textView.setText(id);

    }
}

File: B8024bind/app/src/main/java/com/example/b8024bind/MainActivity.kt
package com.example.b8024bind

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.util.TypedValue
import com.example.b8024bind.databinding.ActivityMainBinding

class MainActivity : AppCompatActivity() {

    private lateinit var binding: ActivityMainBinding

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        binding = ActivityMainBinding.inflate(layoutInflater)
        setContentView(binding.root)
        binding.textView.setTextSize(TypedValue.COMPLEX_UNIT_SP, 31.5f)
        binding.textView.setText("olleH")
    }

    override fun onDestroy() {
        super.onDestroy()
        binding = null
    }
}